onadata/apps/api/management/commands/retrieve_org_or_project_list.py

```diff
@@ -0,0 +1,133 @@
+import json
+from django.core.management.base import (
+    BaseCommand, CommandError, CommandParser)
+from django.utils.translation import gettext as _
+
+from onadata.apps.logger.models import Project
+from onadata.apps.api.models import OrganizationProfile
+from onadata.libs.permissions import is_organization, get_role
+from onadata.libs.serializers.organization_serializer import \
+    OrganizationSerializer
+
+
+def get_project_users(project):
+    ret = {}
```

Comment:
The above two functions looks like good candidates to be in a util file and can be reused in other places